google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -166,7 +170,7 @@ def run(self):
 
         autoflush = self.session.autoflush
         try:
-            self.session.autoflush = False
+            self.session.autoflush = True
```

Comment:
question: Why is the change here from manual flush to autoflush?  Is there any side-effect?  Should there be a comment here to document why this is `True` so that people don't change this in the future?

Reply:
Updated with comment